doc/source/parameters/cfd/ale.rst

```diff
@@ -8,20 +8,22 @@ Lethe contains a small Arbitrary Lagrangian-Eulerian (ALE) module which enables
 .. code-block:: text
 
    subsection ALE
-    set enable                = false
+    set enable = false
 
-    subsection uvw
+    subsection velocity
       set Function expression = 0; 0 # In 2D : u;v
-        or
+      # or
       set Function expression = 0; 0; 0 #In 3D u;v;w
     end
    
    end
 
-* The ``enable`` parameter is set to true if the ALE velocity should be substracted from the fluid velocity.
+* The ``enable`` parameter is set to ``true`` if the ALE velocity should be subtracted from the fluid velocity.
+
+The ALE velocity field is specified through the ``velocity`` subsection.
 
 * The ``Function expression`` parameter sets the expression for the ALE velocity field in regards to :math:`u` and :math:`v`  for a 2D simulation and to :math:`u`, :math:`v`, :math:`w` for a 3D simulation.
 
-.. warning:: The current implementation of the ALE module does not support time-dependent or non-divergence velocity fields
+.. warning:: The current implementation of the ALE module does not support time-dependent or divergence-free velocity fields.
```

Comment:
```suggestion
.. warning:: The current implementation of the ALE module does not support time-dependent or non divergence-free velocity fields.
```
The velocity have to be divergence free :)

Reply:
You're right! ^^'